networking/network_policy/deleting-network-policy.adoc

```diff
@@ -1,11 +1,19 @@
 :_content-type: ASSEMBLY
 [id="deleting-network-policy"]
 = Deleting a network policy
+ifdef::openshift-enterprise[]
```

Comment:
The same comments apply here regarding the attribute files. Please see my preceding comment for details.